io.github.kotatogram.json

```diff
@@ -0,0 +1,163 @@
+{
+    "id": "io.github.kotatogram",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "kotatogram-desktop",
+    "rename-icon": "kotatogram",
+    "finish-args": [ "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=wayland",
+                     "--socket=pulseaudio",
+                     "--talk-name=org.kde.StatusNotifierWatcher",
+                     "--talk-name=org.freedesktop.Notifications",
+                     "--talk-name=com.canonical.Unity",
+                     "--talk-name=org.freedesktop.portal.Fcitx",
+                     "--filesystem=xdg-download",
+                     "--device=dri" ],
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "add-ld-path": ".",
+            "version": "19.08",
+            "autodownload": true,
+            "autodelete": false
+        }
+    },
+    "cleanup-commands": [
+        "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
+    ],
+    "cleanup": [ "/cache",
+                 "/man",
+                 "/share/man",
+                 "/lib/systemd",
+                 "/include",
+                 "/lib/pkgconfig",
+                 "/lib/cmake",
+                 "*.la", "*.a"],
+    "modules": [
+        {
+            "name": "cmake",
```

Comment:
Any reason of rebuilding cmake here?

Reply:
tdesktop/kptatogram-desktop require cmake 3.16